examples/allennlp/allennlp_simple.py

```diff
@@ -33,38 +34,33 @@
 import optuna
 from optuna.integration import AllenNLPPruningCallback
 
+sys.path.append(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
Trick for importing `subsample_dataset_reader` when this example is launched on the top of this repo.

Reply:
Hmm, this trick does not work when users use CLI. I got the following error:
```console
name '__file__' is not defined
Traceback (most recent call last):
  File "/Users/yanase/.pyenv/versions/optuna-dev/lib/python3.7/site-packages/cliff/app.py", line 400, in run_subcommand
    result = cmd.run(parsed_args)
  File "/Users/yanase/.pyenv/versions/optuna-dev/lib/python3.7/site-packages/cliff/command.py", line 187, in run
    return_code = self.take_action(parsed_args) or 0
  File "/Users/yanase/pfn/code/optuna/optuna/cli.py", line 325, in take_action
    loader.exec_module(target_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "allennlp_simple.py", line 37, in <module>
    sys.path.append(os.path.dirname(os.path.abspath(__file__)))
NameError: name '__file__' is not defined
```

Reply:
Oh, you're right. Based on #1394, I'm wondering if I could remove a description of `optuna CLI` here.
What do you think?

Reply:
That makes sense. I think we can remove the instruction about `optuna study optimize`.

Reply:
ac4a13c removed the document about `optuna study optimize`.
I also remove it about `python allennlp_simple.py` as it is obvious to run a script,
after removing the way to use optuna cli.

Reply:
> I also remove it about python allennlp_simple.py as it is obvious to run a script

Sounds good! The change is aligned with #1566.